skeleton/package.json

```diff
@@ -18,7 +18,7 @@
     "test:routes": "jest -c jest.routes.config.js",
     "test:a11y": "jest --detectOpenHandles --forceExit -c jest.a11y.config.js --maxWorkers 15",
     "test:smoke": "NODE_TLS_REJECT_UNAUTHORIZED=0 jest -c jest.smoke.config.js",
-    "test:functional": "NODE_TLS_REJECT_UNAUTHORIZED=0 codeceptjs run --steps",
+    "test:functional": "NODE_TLS_REJECT_UNAUTHORIZED=0 yarn playwright install && codeceptjs run --steps",
```

Comment:
Can you make sure to create a new app from this template to ensure this works? Not sure if this should be required

Reply:
The reason I made this change was because Playwright functional tests wouldn't run after I create my lab app:
https://build.hmcts.net/job/HMCTS_j_to_z/job/labs-dawidstrozak-nodejs/view/change-requests/job/PR-9/
```
14:49:26  Initial Functional test --
14:49:26  
14:49:26    ✖ "before all" hook: codeceptjs.beforeSuite for "The home page loads" in 6ms
14:49:26  Error: browserType.launch: Executable doesn't exist at /home/jenkinsssh/.cache/ms-playwright/chromium-1129/chrome-linux/chrome
14:49:26  ╔═════════════════════════════════════════════════════════════════════════╗
14:49:26  ║ Looks like Playwright Test or Playwright was just installed or updated. ║
14:49:26  ║ Please run the following command to download new browsers:              ║
14:49:26  ║                                                                         ║
14:49:26  ║     yarn playwright install                                             ║
14:49:26  ║                                                                         ║
14:49:26  ║ <3 Playwright Team                                                      ║
14:49:26  ╚═════════════════════════════════════════════════════════════════════════╝
```
This ensures playwright is installed/initialised before functional tests are run and if they already are it does nothing AFAIK.
I will create another app locally to make sure it doesn't cause any other issues/side-effects and report back.

Reply:
Creation of a new Node JS app using backstage now works after merging this and the above change is fine.
Tested with a fresh app
https://build.hmcts.net/job/HMCTS_j_to_z/job/labs-dawidstrozak-fixtest-nodejs/job/master/3/